examples/ec2/transitgatewayvpcattachmentaccepter.yaml

```diff
@@ -12,3 +12,48 @@ spec:
     transitGatewayAttachmentIdSelector:
       matchLabels:
         testing.upbound.io/example-name: example
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: TransitGatewayVPCAttachment
+metadata:
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    region: us-west-1
+    subnetIdRefs:
+      - name: sample-subnet1
+    transitGatewayId: tgw-07a4690db599a7263
```

Comment:
This TransitGateway ID was hardcoded. Because, cross-account resource sharing, Transit Gateway resources need access through RAM. This requirement was provided manually in the console for tgw-07a4690db599a7263. The RAM stuff gives access to the peer account to read the tgw-07a4690db599a7263 located in the default account.